applications/ParticleMechanicsApplication/particle_mechanics_application_variables.cpp

```diff
@@ -90,4 +90,6 @@ namespace Kratos
     KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( NODAL_MOMENTUM)
     KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( NODAL_INERTIA)
 
+    // Solver related variables
+    KRATOS_CREATE_VARIABLE(bool, IMPLICIT_IS_LINEAR)
```

Comment:
Does the Variable need to involve IMPLICIT? Isn't GEOMETRICAL_LINEAR enough?

Reply:
Updated